### ISSUE TITLE
Stuck at Load Models 

### ISSUE THREAD

**Issue by syddharth** [sberbank-ai/sber-swap#34]
Stuck at:
input mean and std: 127.5 127.5
find model: ./insightface_func/models\antelope\glintr100.onnx recognition
find model: ./insightface_func/models\antelope\scrfd_10g_bnkps.onnx detection
set det-size: (640, 640)
loading ./coordinate_reg/model/2d106det 0
input mean and std: 127.5 127.5
find model: ./insightface_func/models\antelope\glintr100.onnx recognition
find model: ./insightface_func/models\antelope\scrfd_10g_bnkps.onnx detection
set det-size: (640, 640)

Tried both the inference.py and SberSwapInference.ipynb

I am on a windows machine with RTX 3070 ti

**Comment by syddharth** [comment]
packages in environment

backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backports                 1.0                        py_2    conda-forge
backports.functools_lru_cache 1.6.4              pyhd8ed1ab_0    conda-forge
blas                      1.0                         mkl
brotlipy                  0.7.0           py37h2bbff1b_1003
ca-certificates           2021.10.8            h5b45459_0    conda-forge
certifi                   2021.10.8        py37h03978a9_2    conda-forge
cffi                      1.15.0           py37h2bbff1b_1
chardet                   3.0.4                    pypi_0    pypi
charset-normalizer        2.0.12                   pypi_0    pypi
click                     8.1.2                    pypi_0    pypi
colorama                  0.4.4              pyh9f0ad1d_0    conda-forge
cryptography              36.0.0           py37h21b164f_0
cudatoolkit               11.3.1               h59b6b97_2
cycler                    0.11.0                   pypi_0    pypi
debugpy                   1.5.1            py37hf2a7229_0    conda-forge
decorator                 5.1.1              pyhd8ed1ab_0    conda-forge
dill                      0.3.4                    pypi_0    pypi
distro                    1.7.0                    pypi_0    pypi
docker-pycreds            0.4.0                    pypi_0    pypi
easydict                  1.9                      pypi_0    pypi
eigen                     3.3.7                h59b6b97_1
entrypoints               0.4                pyhd8ed1ab_0    conda-forge
flatbuffers               2.0                      pypi_0    pypi
freetype                  2.10.4               hd328e21_0
future                    0.18.2                   pypi_0    pypi
git                       2.34.1               haa95532_0
gitdb                     4.0.9                    pypi_0    pypi
gitpython                 3.1.27                   pypi_0    pypi
glib                      2.69.1               h5dc1a3c_1
gst-plugins-base          1.18.5               h9e645db_0
gstreamer                 1.18.5               hd78058f_0
hdf5                      1.10.6               h7ebc959_0
icc_rt                    2019.0.0             h0cc432a_1
icu                       58.2                 ha925a31_3
idna                      2.6                      pypi_0    pypi
imageio                   2.15.0                   pypi_0    pypi
importlib-metadata        4.11.3                   pypi_0    pypi
insightface               0.2.1                    pypi_0    pypi
intel-openmp              2021.4.0          haa95532_3556
ipykernel                 6.12.1           py37h90c5f73_0    conda-forge
ipython                   7.32.0           py37h03978a9_0    conda-forge
jedi                      0.18.1           py37h03978a9_1    conda-forge
joblib                    1.1.0                    pypi_0    pypi
jpeg                      9d                   h2bbff1b_0
jupyter_client            7.2.2              pyhd8ed1ab_1    conda-forge
jupyter_core              4.9.2            py37h03978a9_0    conda-forge
kiwisolver                1.4.2                    pypi_0    pypi
kornia                    0.5.4                    pypi_0    pypi
libffi                    3.4.2                h604cdb4_1
libiconv                  1.15                 h1df5818_7
libogg                    1.3.5                h2bbff1b_1
libpng                    1.6.37               h2a8f88b_0
libprotobuf               3.5.1                he0781b1_0
libsodium                 1.0.18               h8d14728_1    conda-forge
libtiff                   4.2.0                hd0e1b90_0
libuv                     1.40.0               he774522_0
libvorbis                 1.3.7                he774522_0
libwebp                   1.2.2                h2bbff1b_0
libwebp-base              1.2.2                h2bbff1b_0
lz4-c                     1.9.3                h2bbff1b_1
matplotlib                3.4.3                    pypi_0    pypi
matplotlib-inline         0.1.3              pyhd8ed1ab_0    conda-forge
mkl                       2021.4.0           haa95532_640
mkl-service               2.4.0            py37h2bbff1b_0
mkl_fft                   1.3.1            py37h277e83a_0
mkl_random                1.2.2            py37hf11a4ad_0
mxnet-cu101               1.5.0                    pypi_0    pypi
mxnet-cu101mkl            1.5.0                    pypi_0    pypi
nest-asyncio              1.5.5              pyhd8ed1ab_0    conda-forge
networkx                  2.6.3                    pypi_0    pypi
numpy                     1.21.5                   pypi_0    pypi
onnx                      1.9.0                    pypi_0    pypi
onnxruntime               1.9.0                    pypi_0    pypi
onnxruntime-gpu           1.11.0                   pypi_0    pypi
opencv                    4.5.4            py37h22b9916_3
opencv-python             4.5.5.64                 pypi_0    pypi
openssl                   1.1.1n               h8ffe710_0    conda-forge
packaging                 21.3               pyhd8ed1ab_0    conda-forge
parso                     0.8.3              pyhd8ed1ab_0    conda-forge
pathtools                 0.1.2                    pypi_0    pypi
pcre                      8.45                 hd77b12b_0
pickleshare               0.7.5                   py_1003    conda-forge
pillow                    9.1.0                    pypi_0    pypi
pip                       21.2.4           py37haa95532_0
promise                   2.3                      pypi_0    pypi
prompt-toolkit            3.0.29             pyha770c72_0    conda-forge
protobuf                  3.20.0                   pypi_0    pypi
psutil                    5.9.0            py37hcc03f2d_1    conda-forge
pycparser                 2.21               pyhd3eb1b0_0
pygments                  2.11.2             pyhd8ed1ab_0    conda-forge
pyopenssl                 22.0.0             pyhd3eb1b0_0
pyparsing                 3.0.7              pyhd8ed1ab_0    conda-forge
pysocks                   1.7.1                    py37_1
python                    3.7.13               h6244533_0
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python-graphviz           0.8.4                    pypi_0    pypi
python_abi                3.7                     2_cp37m    conda-forge
pytorch-mutex             1.0                        cuda    pytorch
pywavelets                1.1.1                    pypi_0    pypi
pywin32                   303              py37hcc03f2d_0    conda-forge
pyyaml                    6.0                      pypi_0    pypi
pyzmq                     22.3.0           py37hcce574b_2    conda-forge
qt                        5.9.7            vc14h73c81de_0
requests                  2.18.4                   pypi_0    pypi
scikit-build              0.14.1                   pypi_0    pypi
scikit-image              0.18.3                   pypi_0    pypi
scikit-learn              1.0.2                    pypi_0    pypi
scipy                     1.7.3                    pypi_0    pypi
sentry-sdk                1.5.8                    pypi_0    pypi
setproctitle              1.2.2                    pypi_0    pypi
setuptools                58.0.4           py37haa95532_0
shortuuid                 1.0.8                    pypi_0    pypi
six                       1.16.0             pyhd3eb1b0_1
smmap                     5.0.0                    pypi_0    pypi
sqlite                    3.38.2               h2bbff1b_0
threadpoolctl             3.1.0                    pypi_0    pypi
tifffile                  2021.11.2                pypi_0    pypi
tk                        8.6.11               h2bbff1b_0
torch                     1.8.1+cu111              pypi_0    pypi
torchaudio                0.11.0               py37_cu113    pytorch
torchvision               0.9.1+cu111              pypi_0    pypi
tornado                   6.1              py37hcc03f2d_3    conda-forge
tqdm                      4.64.0                   pypi_0    pypi
traitlets                 5.1.1              pyhd8ed1ab_0    conda-forge
typing_extensions         4.1.1              pyh06a4308_0
urllib3                   1.22                     pypi_0    pypi
vc                        14.2                 h21ff451_1
vs2015_runtime            14.27.29016          h5e58377_2
wandb                     0.12.14                  pypi_0    pypi
wcwidth                   0.2.5              pyh9f0ad1d_2    conda-forge
wheel                     0.37.1             pyhd3eb1b0_0
win_inet_pton             1.1.0            py37haa95532_0
wincertstore              0.2              py37haa95532_2
xz                        5.2.5                h62dcd97_0
zeromq                    4.3.4                h0e60522_1    conda-forge
zipp                      3.8.0                    pypi_0    pypi
zlib                      1.2.11               hbd8134f_5
zstd                      1.4.9                h19a0ad4_0

**Comment by FBAdmirer** [comment]
> Stuck at: input mean and std: 127.5 127.5 find model: ./insightface_func/models\antelope\glintr100.onnx recognition find model: ./insightface_func/models\antelope\scrfd_10g_bnkps.onnx detection set det-size: (640, 640) loading ./coordinate_reg/model/2d106det 0 input mean and std: 127.5 127.5 find model: ./insightface_func/models\antelope\glintr100.onnx recognition find model: ./insightface_func/models\antelope\scrfd_10g_bnkps.onnx detection set det-size: (640, 640)
> 
> Tried both the inference.py and SberSwapInference.ipynb
> 
> I am on a windows machine with RTX 3070 ti

*******************************************************
I was able to compile mxnet-1.9.0 with CUDA 11.4 (CUDNN v8.2.4.15) and able to install and make it work in Windows 11 system with AMD Ryzen 9 processor. Graphics card is RTX 3070 Ti. 

Attached is the mxnet-1.9.0 windows wheel zip file which needs to be installed with command pip install mxnet-1.9.0-py3-none-any.whl after unzipping.

Conda Environment: Python 3.7

Installed cudatoolkit with conda install -c conda-forge cudatoolkit=11.4
Installed cudnn with conda install -c conda-forge cudnn=8.2
Installed torch with pip install torch==1.11.0+cu113 -f https://download.pytorch.org/whl/torch_stable.html
Installed torchvision with pip install torchvision==0.12.0+cu113 -f https://download.pytorch.org/whl/torch_stable.html
Installed mxnet-cu112 with pip install mxnet-cu112 (could not find mxnet-cu112 for windows)
Installed onnx with pip install onnx==1.8.0
Installed onnxruntime-gpu with pip install onnxruntime-gpu==1.8.0
Installed kornia with pip install kornia==0.5.4
[mxnet-1.9.0-py3-none-any.whl.zip](https://github.com/ai-forever/sber-swap/files/8507368/mxnet-1.9.0-py3-none-any.whl.zip)
